src/Intercom/Clients/ConversationsClient.cs

```diff
@@ -52,5 +52,31 @@ public Conversation View(String id, bool? displayAsPlainText = null)
             result = Get<Conversation>(resource: CONVERSATIONS_RESOURCE + Path.DirectorySeparatorChar + id);
             return result.Result;
         }
+
+        public Conversations ListAll ()
+        {
+            ClientResponse<Conversations> result = null;
+            result = Get<Conversations>(resource: CONVERSATIONS_RESOURCE, parameters: null);
+            return result.Result;
+        }
+
+        public Conversations ListAll(Dictionary<String, String> parameters)
+        {
+            if (parameters == null)
+            {
+                throw new ArgumentNullException(nameof(parameters));
+            }
+
+            if (!parameters.ContainsKey("order") &&
+                !parameters.ContainsKey("sort") &&
+                !parameters.ContainsKey("before"))
+            {
+                throw new ArgumentException(String.Format("'parameters' argument should include at least order, or sort, or before parameter."));
```

Comment:
does it actually need to?

Reply:
Oh, you're right! I misread our docs 🤦‍♂️ Going to change that!